apps/studio/package.json

```diff
@@ -32,95 +33,59 @@
         "coverage": "bun test --coverage",
         "lint": "eslint --fix .",
         "format": "prettier --write .",
-        "increment_tag": "npm version patch",
-        "publish_tag": "./scripts/publish_tag.sh",
-        "remove_tag": "./scripts/remove_tag.sh"
+        "typecheck": "tsc --noEmit"
     },
     "dependencies": {
         "@anthropic-ai/sdk": "^0.29.2",
         "@emotion/react": "^11.13.3",
         "@emotion/styled": "^11.13.0",
         "@fontsource-variable/inter": "^5.1.0",
-        "@onlook/utils": "0.0.10",
-        "@radix-ui/react-accordion": "^1.2.0",
-        "@radix-ui/react-alert-dialog": "^1.1.1",
-        "@radix-ui/react-checkbox": "^1.1.2",
-        "@radix-ui/react-context-menu": "^2.2.1",
-        "@radix-ui/react-dialog": "^1.1.1",
-        "@radix-ui/react-dropdown-menu": "^2.1.1",
-        "@radix-ui/react-icons": "^1.3.0",
-        "@radix-ui/react-label": "^2.1.0",
-        "@radix-ui/react-popover": "^1.1.1",
-        "@radix-ui/react-progress": "^1.1.0",
-        "@radix-ui/react-scroll-area": "^1.2.0",
-        "@radix-ui/react-select": "^2.1.1",
-        "@radix-ui/react-separator": "^1.1.0",
-        "@radix-ui/react-slot": "^1.1.0",
-        "@radix-ui/react-tabs": "^1.1.0",
-        "@radix-ui/react-toast": "^1.2.1",
-        "@radix-ui/react-toggle": "^1.1.0",
-        "@radix-ui/react-toggle-group": "^1.1.0",
-        "@radix-ui/react-tooltip": "^1.1.2",
+        "@onlook/foundation": "*",
```

Comment:
Let me know if I'm wrong here but wouldn't the syntax be "workspace:*" ?

https://bun.sh/guides/install/workspaces